README.rst

```diff
@@ -3,7 +3,7 @@ p5
 
 .. raw:: html
 
-   <p align="center">
+   <p align="left">
```

Comment:
I believe you can achieve the same effect without `.. raw::html` using the following syntax
```
.. image:: <image link>
    :alt: <alt, optional>
    :target: <link to redirect the user to, optional>
```

Reply:
Let me try it

Reply:
I have tried it but they do not stay in a row. They are in a column otherwise.

Reply:
Worked !!!